src/ui/components/TurnControl/TurnControl.tsx

```diff
@@ -123,8 +126,15 @@ export const TurnControl = ({ match }: TurnControlProps) => {
   const { [sessionOwnerPlayerId]: sessionOwnerPlayer, ...opponents } =
     match.table.players
 
+  if (!sessionOwnerPlayer) {
+    throw new Error('Session owner player not found')
```

Comment:
This would only happen if the match state was corrupt, so this should be a `MatchStateCorruptError`:
```suggestion
    throw new MatchStateCorruptError('Session owner player not found')
```